cellxgene_resources/curation_qa.ipynb

```diff
@@ -455,20 +446,10 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "prop = 'sample_id'\n",
```

Comment:
We want to keep this general feature of checking barcodes per obs_field because it's the sample_id, donor_id, etc. that will likely be one assay or another